scm/driver/bitbucket/webhook.go

```diff
@@ -138,6 +141,12 @@ func (s *webhookService) parsePullRequestHook(data []byte) (*scm.PullRequestHook
 	}
 }
 
+func (s *webhookService) parsePipelineHook(data []byte) (*scm.PipelineHook, error) {
+	dst := new(pipelineHook)
+	err := json.Unmarshal(data, dst)
+	return convertBitbucketHook(dst), err
+}
```

Comment:
error handling seems to be missing, please check

Reply:
Fixed